doc/setup.md

```diff
@@ -165,3 +165,18 @@ make load_demo_data
 ```
 
 This also adds `testuser@cesium-ml.org` as an administrator.
+
+### Deploying secure HTTP / SSL certificate
+
+When running a public server, you will likely want to deploy an SSL certificate (i.e., serve `https://your.url` instead of `http://your.url`). Certificates can be obtained for free from services such as Let's Encrypt (https://letsencrypt.org/). For Let's Encrypt at least, the process requires verification of the service without SSL first to provision a certificate, and so deployment without an nginx configuration change will be first required.
```

Comment:
```suggestion
When running a public server, you will likely want to deploy an SSL certificate (i.e., serve `https://your.url` instead of `http://your.url`). Certificates can be obtained for free from services such as Let's Encrypt (https://letsencrypt.org/). For Let's Encrypt at least, the process requires verification of the service without SSL first.
```

Reply:
OK, so what does the user have to do to kick off that process?